app/queue/event_producer.py

```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+
+from kafka import KafkaProducer
+
+from app.logging import get_logger
+from app.queue import metrics
+
+logger = get_logger(__name__)
+
+Topics = Enum("Topics", ("egress", "event"))
```

Comment:
Let’s make this singular.

```suggestion
Topic = Enum("Topic", ("egress", "event"))
```

I even considered moving this to _app.config_, but here it’s probably fine as this is the module that deals with topics.

Reply:
I changed it anyway, it was just a few spots. It seemed worth it to just fix it now even if it wasn't a huge deal